### PR TITLE
ztp: update README and validator policy examples

### PR DESCRIPTION
**Pull request by Missxiaoguo**

### CODE REVIEW

File: ztp/gitops-subscriptions/argocd/README.md [hunk 1]
```diff
@@ -325,3 +325,19 @@ Check labels on MangedCluster:
...
 ```
 If the Namespace, OperatorGroup, and Subscription policies are compliant but the operator configuration policies are not it is likely that the operators did not install on the spoke cluster. This causes the operator config policies to fail to apply because the CRD is not yet applied to spoke.
+
+### Restart Policies reconciliation
+A ClusterGroupUpgrade CR is generated in the namespace ```ztp-install``` by Topology Aware Lifecycle Operator after the managed spoke cluster becomes ```Ready```:
+```
+     $ export CLUSTER=<clusterName>
+     $ oc get clustergroupupgrades -n ztp-install $CLUSTER
+```
+If unexpected issues happen that cause the Policies fail to become complaint within the configured timeout(default is 4h), the status of ClusterGroupUpgrade CR will show ```UpgradeTimedOut```:
```
Comment (irinamihai) [inline ztp/gitops-subscriptions/argocd/README.md:335]: I would maybe rephrase to: "If there are unexpected issues and the Policies fail to become compliant within..."

File: ztp/gitops-subscriptions/argocd/README.md [hunk 2]
```diff
@@ -325,3 +325,19 @@ Check labels on MangedCluster:
     $ oc get policy -n $CLUSTER
 ```
 If the Namespace, OperatorGroup, and Subscription policies are compliant but the operator configuration policies are not it is likely that the operators did not install on the spoke cluster. This causes the operator config policies to fail to apply because the CRD is not yet applied to spoke.
+
+### Restart Policies reconciliation
+A ClusterGroupUpgrade CR is generated in the namespace ```ztp-install``` by Topology Aware Lifecycle Operator after the managed spoke cluster becomes ```Ready```:
```
Comment (irinamihai) [inline ztp/gitops-subscriptions/argocd/README.md:330]: by the

File: ztp/gitops-subscriptions/argocd/README.md [hunk 3]
```diff
@@ -325,3 +325,19 @@ Check labels on MangedCluster:
...
+```
+     $ export CLUSTER=<clusterName>
+     $ oc get clustergroupupgrades -n ztp-install $CLUSTER
+```
+If unexpected issues happen that cause the Policies fail to become complaint within the configured timeout(default is 4h), the status of ClusterGroupUpgrade CR will show ```UpgradeTimedOut```:
+```
+     $ oc get clustergroupupgrades -n ztp-install $CLUSTER -o jsonpath='{.status.conditions[?(@.type=="Ready")]}'
+```
+A retry of policies reconciliation will be triggered after 1h when ```UpgradeTimedOut``` happens. Once you have investigated/fixed the issues, you can trigger the reconciliation immediately by deleting the existing ClusterGroupUpgrade CR, this will make a new ClusterGroupUpgrade CR to be created automatically:
```
Comment (irinamihai) [inline ztp/gitops-subscriptions/argocd/README.md:339]: Suggestion to rephrase at the end:
". This will automatically trigger the creation of a new ClusterGroupUpgrade CR."

File: ztp/gitops-subscriptions/argocd/README.md [hunk 4]
```diff
@@ -325,3 +325,19 @@ Check labels on MangedCluster:
...
+```
+     $ oc delete clustergroupupgrades -n ztp-install $CLUSTER
+```
+Please note that once ClusterGroupUpgrade CR completes with status ```UpgradeCompleted``` and the managed spoke cluster has label ```ztp-done``` applied, if you would like to make additional configuration via PGT, deleting the existing ClusterGroupUpgrade CR will not make TALO generating a new CR, you have the responsibility to create the ClusterGroupUpgrade CR.
```
Comment (irinamihai) [inline ztp/gitops-subscriptions/argocd/README.md:343]: "will not make TALO generate"

File: ztp/gitops-subscriptions/argocd/README.md [hunk 5]
```diff
@@ -325,3 +325,19 @@ Check labels on MangedCluster:
...
+A ClusterGroupUpgrade CR is generated in the namespace ```ztp-install``` by the Topology Aware Lifecycle Operator after the managed spoke cluster becomes ```Ready```:
+```
+     $ export CLUSTER=<clusterName>
+     $ oc get clustergroupupgrades -n ztp-install $CLUSTER
+```
+If there are unexpected issues and the Policies fail to become complaint within the configured timeout(default is 4h), the status of ClusterGroupUpgrade CR will show ```UpgradeTimedOut```:
+```
+     $ oc get clustergroupupgrades -n ztp-install $CLUSTER -o jsonpath='{.status.conditions[?(@.type=="Ready")]}'
+```
+A retry of policies reconciliation will be triggered after 1h when ```UpgradeTimedOut``` happens. Once you have investigated/fixed the issues, you can trigger the reconciliation immediately by deleting the existing ClusterGroupUpgrade CR. This will automatically trigger the creation of a new ClusterGroupUpgrade CR.
```
Comment (lack) [inline ztp/gitops-subscriptions/argocd/README.md:339]:
```suggestion
A ClusterGroupUpgrade CR in the `UpgradeTimedOut` state will automatically restart its policy reconciliation every 1h. If you have changed your policies you can start a retry immediately by deleting the existing ClusterGroupUpgrade CR. This will trigger the automatic creation of a new ClusterGroupUpgrade CR which begins reconciling the policies immediately:
```

File: ztp/gitops-subscriptions/argocd/README.md [hunk 6]
```diff
@@ -325,3 +325,19 @@ Check labels on MangedCluster:
...
 ```
 If the Namespace, OperatorGroup, and Subscription policies are compliant but the operator configuration policies are not it is likely that the operators did not install on the spoke cluster. This causes the operator config policies to fail to apply because the CRD is not yet applied to spoke.
+
+### Restart Policies reconciliation
+A ClusterGroupUpgrade CR is generated in the namespace ```ztp-install``` by the Topology Aware Lifecycle Operator after the managed spoke cluster becomes ```Ready```:
+```
+     $ export CLUSTER=<clusterName>
+     $ oc get clustergroupupgrades -n ztp-install $CLUSTER
+```
+If there are unexpected issues and the Policies fail to become complaint within the configured timeout(default is 4h), the status of ClusterGroupUpgrade CR will show ```UpgradeTimedOut```:
```
Comment (lack) [inline ztp/gitops-subscriptions/argocd/README.md:335]:
```suggestion
If there are unexpected issues and the Policies fail to become complaint within the configured timeout(default is 4h), the status of the ClusterGroupUpgrade CR will show ```UpgradeTimedOut```:
```

File: ztp/gitops-subscriptions/argocd/README.md [hunk 7]
```diff
@@ -325,3 +325,19 @@ Check labels on MangedCluster:
...
+```
+     $ oc get clustergroupupgrades -n ztp-install $CLUSTER -o jsonpath='{.status.conditions[?(@.type=="Ready")]}'
+```
+A retry of policies reconciliation will be triggered after 1h when ```UpgradeTimedOut``` happens. Once you have investigated/fixed the issues, you can trigger the reconciliation immediately by deleting the existing ClusterGroupUpgrade CR. This will automatically trigger the creation of a new ClusterGroupUpgrade CR.
+```
+     $ oc delete clustergroupupgrades -n ztp-install $CLUSTER
+```
+Please note that once ClusterGroupUpgrade CR completes with status ```UpgradeCompleted``` and the managed spoke cluster has label ```ztp-done``` applied, if you would like to make additional configuration via PGT, deleting the existing ClusterGroupUpgrade CR will not make TALO generate a new CR, you have the responsibility to create the ClusterGroupUpgrade CR.
```
Comment (lack) [inline ztp/gitops-subscriptions/argocd/README.md:343]:
```suggestion
Please note that once ClusterGroupUpgrade CR completes with status ```UpgradeCompleted``` and the managed spoke cluster has label ```ztp-done``` applied, if you would like to make additional configuration via PGT, deleting the existing ClusterGroupUpgrade CR will not make TALO generate a new CR.  At this point ZTP has completed its interaction with the cluster and any further interactions should be treated as an upgrade. See the [Topology-Aware Lifecycle Operator](https://github.com/openshift-kni/cluster-group-upgrades-operator#readme) documentation for instructions on how to construct your own ClusterGroupUpgrade CR to apply the new canges.
```

**Review by lack — CHANGES_REQUESTED** [review]

File: ztp/gitops-subscriptions/argocd/README.md (hunk 7 above)
Comment (Missxiaoguo) [reply ztp/gitops-subscriptions/argocd/README.md]: Nice suggestion. Thank you :)

**Review by lack — APPROVED** [review]
